tf-psa-crypto/scripts/generate_driver_wrappers.py

```diff
@@ -179,6 +179,8 @@ def main() -> int:
     args = parser.parse_args()
 
     project_root = os.path.abspath(args.project_root)
+    if build_tree.looks_like_mbedtls_root(project_root):
```

Comment:
I take it that is to allow calling the scripts from the library root as well as the new location?

Reply:
We need to keep supporting the call from mbedtls for the make build system.